src/content/docs/apis/nerdgraph/examples/nerdgraph-user-mgmt.mdx

```diff
@@ -1,41 +1,41 @@
 ---
 
-You can use New Relic's [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to view and manage user groups and access grants. For how to do this in the UI, see the [user management UI docs](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks). 
+You can use New Relic's [NerdGraph API](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph) to view and manage user groups and what those groups can access. For how to do this in the UI, see the [user management UI docs](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-ui-and-tasks). 
 
 ## Requirements [#requirements]
 
 Some requirements for managing users and groups via NerdGraph: 
-* Requires [Pro or Enterprise edition](https://newrelic.com/pricing) (Standard edition organizations can't configure groups or access grants). 
+* Requires [Pro or Enterprise edition](https://newrelic.com/pricing) (Standard edition organizations can't configure groups or roles). 
 * If you're using [SCIM provisioning](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign): for that authentication domain, you can't create groups or add users to groups, because your groups and users are managed via SCIM.   
 * You must be a user on [our newer user model](/docs/accounts/original-accounts-billing/original-users-roles/overview-user-models#determine-user-model). Other permissions-related requirements: 
   * Required [user type](/docs/accounts/accounts-billing/new-relic-one-user-management/user-type): core user or full platform user 
-  * Required [role](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#roles): **Organization manager** or **Authentication domain manager**
+  * Required [administration settings](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#admin-settings): **Organization settings** or **Authentication domain settings**
 
 ## Before you start [#concepts]
 
 Before using NerdGraph to manage users: 
 
 * Ensure you have a decent understanding of our [user management concepts](/docs/accounts/accounts-billing/new-relic-one-user-management/user-management-concepts#understand-concepts)
-* If you haven't already, it can help to look in the **Access management** UI to improve your understanding of how groups and access grants work, and to understand any access grants that already exist. 
-* Have a good plan for the groups and access grants you need to create. It may help to map this out in a spreadsheet beforehand. 
+* If you haven't already, it can help to look in the **Access management** UI to improve your understanding of how groups and user access works, and to understand groups that already exist. 
+* Have a good plan for the group access you need to create. It may help to map this out in a spreadsheet beforehand. 
```

Comment:
For me, I would say something like, "If you haven't already, we suggest looking in the Access management UI... Before you do this, we recommend creating a plan for the group access you need to create..."